src/js/modules/breakpointHelper.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @overview Breakpoint handler
+ *
+ * @module breakpointHelper
+ */
+
+export const getBreakpoints = () => {
```

Comment:
Would be good to comment these functions, so that they have a brief description of what they do and what they return